src/pages/PrivateRoute/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component, Fragment } from 'react';
+import { Grid, GridItem, Button, Title } from '@patternfly/react-core';
+import AuthLayout from '@/components/AuthLayout';
+import { routerRedux } from 'dva/router';
+import { connect } from 'dva';
+import styles from './index.less';
+
+@connect(user => ({
+  user: user.user,
+}))
+class PrivateRoute extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  navigate = page => {
+    const { dispatch } = this.props;
+    dispatch(routerRedux.push(`/${page}`));
+  };
+
+  render() {
+    const { children, user } = this.props;
+    // Basic Login methods
+    const PbenchMethods = (
+      <Grid gutter="md" className={styles.section}>
+        <GridItem>
+          <Button
+            isBlock
+            variant="primary"
+            onClick={() => this.navigate('login')}
+            className={styles.btn}
+          >
+            <Title headingLevel="h4" size="xl" style={{ color: 'white' }}>
+              Pbench credentials
+            </Title>
+          </Button>
+        </GridItem>
+      </Grid>
+    );
+
+    const restLoginHandlers = (
+      <Grid gutter="md" className={styles.restLoginDiv}>
+        <GridItem>
+          <Title headingLevel="h4" size="xl">
+            Need an account?
+            <Button
+              variant="link"
+              onClick={() => this.navigate('signup')}
+              className={styles.inlineLink}
+            >
+              signup
```

Comment:
`Sign up` should be two words and have a capital `S`. We should also consider adding padding to the left of this button as it is currently rendering right next to the `Title` parent element.

Reply:
fixed